.github/workflows/release.yml

```diff
@@ -22,21 +22,24 @@ jobs:
       run: |
         curl -sSL https://install.python-poetry.org | python3 -
         echo "$HOME/.local/bin" >> $GITHUB_PATH
+    
```

Comment:
Keep the consistency file in all the repos. In some repo it's named as build.yaml file